$OpenBSD: patch-storage_mozStorageService_cpp,v 1.1 2020/02/13 07:41:53 landry Exp $

dont check for SQLITE_SECURE_DELETE at runtime, we disable it

Cf https://bugzilla.mozilla.org/show_bug.cgi?id=1049920 & https://phabricator.services.mozilla.com/D18057

Index: storage/mozStorageService.cpp
--- storage/mozStorageService.cpp.orig
+++ storage/mozStorageService.cpp
@@ -188,7 +188,6 @@ already_AddRefed<Service> Service::getSingleton() {
   // or newer.  Our configure check ensures we are using a new enough version
   // at compile time.
   if (SQLITE_VERSION_NUMBER > ::sqlite3_libversion_number() ||
-      !::sqlite3_compileoption_used("SQLITE_SECURE_DELETE") ||
       !::sqlite3_compileoption_used("SQLITE_THREADSAFE=1") ||
       !::sqlite3_compileoption_used("SQLITE_ENABLE_FTS3") ||
       !::sqlite3_compileoption_used("SQLITE_ENABLE_UNLOCK_NOTIFY") ||
